src/Generation/Generator/Renderer/Internal/Errors.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Generator.Model;
+
+namespace Generator.Renderer.Internal;
+
+internal static class Errors
```

Comment:
Rename your `Errors` renderer to `Error`. The other renderers are named in plural form because their `Render` method accepts an `IEnumerable` of the corresponding class / interface. There are other renderer which are named singular, too.